recipes/theora/all/conanfile.py

```diff
@@ -1,182 +1,105 @@
+        # note: MSVC does not support inline assembly for 64 bit builds
+        tc.variables["USE_X86_64"] = (self.settings.arch == "x86_64" and not is_msvc(self))
+        tc.generate()
+        cd = CMakeDeps(self)
+        cd.generate()
 
     def build(self):
-        if self._is_msvc:
-            self._build_msvc()
-        else:
-            shutil.copy(self._user_info_build["gnu-config"].CONFIG_SUB,
-                        os.path.join(self._source_subfolder, "config.sub"))
-            shutil.copy(self._user_info_build["gnu-config"].CONFIG_GUESS,
-                        os.path.join(self._source_subfolder, "config.guess"))
-            configure = os.path.join(self._source_subfolder, "configure")
-            permission = stat.S_IMODE(os.lstat(configure).st_mode)
-            os.chmod(configure, (permission | stat.S_IEXEC))
-            # relocatable shared libs on macOS
-            tools.replace_in_file(configure, "-install_name \\$rpath/", "-install_name @rpath/")
-            # avoid SIP issues on macOS when dependencies are shared
-            if tools.is_apple_os(self.settings.os):
-                libpaths = ":".join(self.deps_cpp_info.lib_paths)
-                tools.replace_in_file(
-                    configure,
-                    "#! /bin/sh\n",
-                    "#! /bin/sh\nexport DYLD_LIBRARY_PATH={}:$DYLD_LIBRARY_PATH\n".format(libpaths),
-                )
-            autotools = self._configure_autotools()
-            autotools.make()
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
 
     def package(self):
-        self.copy(pattern="LICENSE", dst="licenses", src=self._source_subfolder)
-        self.copy(pattern="COPYING", dst="licenses", src=self._source_subfolder)
-        if self._is_msvc:
-            include_folder = os.path.join(self._source_subfolder, "include")
-            self.copy(pattern="*.h", dst="include", src=include_folder)
-            self.copy(pattern="*.dll", dst="bin", keep_path=False)
-            self.copy(pattern="*.lib", dst="lib", keep_path=False)
-        else:
-            autotools = self._configure_autotools()
-            autotools.install()
-            tools.remove_files_by_mask(os.path.join(self.package_folder, "lib"), "*.la")
-            tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
-            tools.rmdir(os.path.join(self.package_folder, "share"))
+        copy(self, "COPYING", src=self.source_folder, dst=os.path.join(self.package_folder, "licenses"))
+        copy(self, "LICENSE", src=self.source_folder, dst=os.path.join(self.package_folder, "licenses"))
+        cmake = CMake(self)
+        cmake.install()
 
     def package_info(self):
-        self.cpp_info.set_property("pkg_config_name", "theora_full_package") # to avoid conflicts with _theora component
+        self.cpp_info.set_property("pkg_config_name", "theora_full_package")    # to avoid conflicts with theora component
```

Comment:
```suggestion
        self.cpp_info.set_property("pkg_config_name", "theora_full_package_do_not_use")    # to avoid conflicts with theora component
```